tools/ui-components/src/toggle-button/toggle-button.tsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import { ButtonSize, ButtonStyle, ToggleButtonProps } from './types';
+
+const defaultClassNames = [
+  'relative',
+  'border-3',
+  'text-center',
+  'inline-block',
+  'cursor-pointer',
+  'active:before:w-full',
+  'active:before:h-full',
+  'active:before:absolute',
+  'active:before:inset-0',
+  'active:before:border-3',
+  'active:before:border-transparent',
+  'active:before:bg-gray-900',
+  'active:before:opacity-20',
+  'focus:outline-none', // Hide the default browser outline
+  'focus:ring',
+  'focus:ring-focus-outline-color',
+  'focus-within:ring',
+  'focus-within:ring-focus-outline-color',
+  'aria-disabled:cursor-not-allowed',
+  'aria-disabled:opacity-50',
+  'ml-[-3px]',
+  'first:ml-0'
+];
+
+const computeClassNames = ({
+  bsSize,
+  bsStyle,
+  checked,
+  disabled
+}: {
+  bsSize: ButtonSize;
+  bsStyle: ButtonStyle;
+  checked?: boolean;
+  disabled?: boolean;
+}) => {
+  const classNames = [...defaultClassNames, checked ? 'cursor-default' : ''];
+
+  switch (bsStyle) {
+    case 'danger':
```

Comment:
Could not find 'danger' usage in the codebase. 
If it is not used, you could omit the switch completely and remove its styles.